examples/Dockerfile

```diff
@@ -20,6 +20,8 @@ shadow \
 procps-ng \
 awk \
 iptables \
+cronie \
+openssh \
```

Comment:
Please include in ansible-security-hardening/roles/cis/tasks/main.yml as your changes are not currently running in build: https://travis-ci.org/vmware/ansible-security-hardening/builds/636311091?utm_source=github_status&utm_medium=notification

Please also add these packages in README.md:

### Prerequisites

* ansible (host)
* python3 (target)
* procps-ng (target)
* awk (target)
* iptables (target)
---here--

#install python3 for ansible
#install shadow for cis rule 6.1.2
#install awk for cis rule 6.1.10
#install procps-ng for cis rule 3.1.1
#install iptables for cis rule 3.6.2
#install photon-release for cis rule 1.7.1.6
tdnf --installroot ~/testroot \
--releasever 3.0 --nogpgcheck \
install python3 shadow awk procps-ng iptables photon-release -y
```
----also above---

Reply:
Adressed

Reply:
Addressed